README.md

```diff
@@ -43,6 +43,21 @@ try {
 }
 ```
 
+You can also use chaining methods if you do not pass the callback function. Check [here](#method-chaining) for details.
+
+```js
+// chaining methods
+inject(dispatch)
+  .get('/')                   // set the request method to GET, and request URL to '/'
```

Comment:
I would write two examples, one with `get` and one with `post`, since this one is not correct (you can't have a body in a `GET` request).